.github/workflows/collector-full.yml

```diff
@@ -138,15 +138,64 @@ jobs:
             -e @'${{ github.workspace }}/ansible/secrets.yml' \
             ansible/ci-build-collector-full.yml
 
+  # ARM doesn't have a full version, so we retag the slim/base image to
+  # take its place
+  retag-arm-image:
+    name: Retag arm slim image
+    runs-on: ubuntu-latest
+    if: |
+      inputs.build-full-image &&
+      !contains(github.event.pull_request.labels.*.name, 'skip-multiarch-builds')
+    env:
+      COLLECTOR_IMAGE: quay.io/stackrox-io/collector:${{ inputs.collector-tag }}
+      RHACS_ENG_IMAGE: quay.io/rhacs-eng/collector:${{ inputs.collector-tag }}
+    steps:
+      - name: Pull image to retag
+        run: |
+          docker pull ${{ env.COLLECTOR_IMAGE }}-arm64-slim
+
+      - name: Retag and push stackrox-io -slim
```

Comment:
The tags for each section `name` are not consistent between stackrox-io and rhacs-eng.